Assets/Scripts/Models/Tile.cs

```diff
@@ -410,7 +409,7 @@ public IEnumerable<ContextMenuAction> GetContextMenuActions(ContextMenu contextM
             {
                 Text = "Cancel Job",
                 RequireCharacterSelected = false,
-                Action = (cm, c) => 
+                Action = (cm, c) =>
```

Comment:
Could we move this actions to methods and assign methods instead of lambda expressions?